relay-general/src/pii/regexes.rs

```diff
@@ -158,6 +159,17 @@ static EMAIL_REGEX: Lazy<Regex> = Lazy::new(|| {
     .unwrap()
 });
 
+static IBAN_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(
+        r#"(?x)
+            \b
+            [A-Z]{2}\d{2}[A-Z0-9]{11,29}
```

Comment:
this regex matches any iban length from the shortest (norway) to the longest (russia). However, it does not check if the country-code matches the length. for example it will scrub an iban with the russian country code but the length of norway, despite that being invalid. 

I figured i'd wanna start out simple and maybe in the future add some additional logic.